docs/csharp/language-reference/operators/multiplication-operator.md

```diff
@@ -16,11 +16,9 @@ author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # * Operator (C# Reference)
-The multiplication operator (`*`), which computes the product of its operands.  Also, the dereference operator, which allows reading and writing to a pointer.  
+The multiplication operator (`*`) computes the multiplicative product of its operands. All numeric types have predefined multiplication operators.  
```

Comment:
*multiplicative* can be removed, as product is about multiplication.
Why did you remove the sentence about dereferencing?

Reply:
At first I thought "hey, this probably belongs in the Remarks section", but there is already a (much better written) sentence in the Remarks section for its usage with pointers.

Reply:
Still it should be mentioned in the first paragraph as well, since Remarks is optional to read and one might skip that section.